Sources/DistributedCluster/Cluster/ClusterControl.swift

```diff
@@ -314,7 +347,7 @@ public struct ClusterControl {
                 throw Cluster.MembershipError(.notFound(node, in: membership))
             }
 
-            if atLeastStatus <= foundMember.status {
```

Comment:
another mini bug; we should error out if the expected status is lower than the current one.

if they are the same, don't error -- we succeeded; the test covers this now